docker/config/default.config.envsubst

```diff
@@ -1,3 +1,5 @@
+api.mesh.enabled=$RADIXDLT_MESH_API_ENABLED
```

Comment:
Can we put this just above `api.mesh.port`?

Reply:
Was really confused when you initially asked to put it in the first line 🤣